package/network/services/dnsmasq/files/dnsmasq.init

```diff
@@ -1066,6 +1066,7 @@ dnsmasq_start()
 			echo "search $DOMAIN" >> /tmp/resolv.conf
 		}
 		DNS_SERVERS="$DNS_SERVERS 127.0.0.1"
+		[ -d /sys/module/ipv6 ] && grep -q 0 /sys/module/ipv6/parameters/disable && DNS_SERVERS="$DNS_SERVERS ::1"
```

Comment:
Why don't just use the same check as in ppp.sh [ -e /proc/sys/net/ipv6 ] && DNS_SERVERS="$DNS_SERVERS ::1"

Reply:
Since this is system wide resolv.conf handling and shouldn't be interface configuration sensitive I elected to use the underlying module configuration instead of the sysctl interface mechanisms. In practice I don't think there's a large difference except for the case where IPv6 is disabled via the `disable` parameter, whether via boot or other module load configuration.

[1] https://www.kernel.org/doc/html/latest/networking/ipv6.html

Reply:
Besides the consistency argument to keep the checks aligned this also uses grep which is suboptimal. Therefore I prefer to use the simple check [ -e /proc/sys/net/ipv6 ]

Reply:
I don't have a strong argument either way, nor do I have a  pressing use case for IPv6 disabled. I'll change and update - any objection to keeping it the `-d` check instead of simply `-e`?

Reply:
I prefer to keep the checks identical to keep the code base consistent